lib/utils.js

```diff
@@ -215,3 +218,12 @@ export const flattenObjectDeep = obj =>
   );
 
 export const stripHTML = htmlContent => sanitizeHtml(htmlContent, { allowedTags: [], allowedAttributes: {} });
+
+export const checkRequiresAddress = values => {
+  return (
+    values.payee &&
+    ![CollectiveType.COLLECTIVE, CollectiveType.EVENT, CollectiveType.PROJECT].includes(values.payee.type) &&
```

Comment:
Add FUND.

Reply:
What about Self Hosted (aka "Independent") Collectives?

They should be treated like an Organization.

Reply:
Fund is added and I've also added a condition to make sure self hosted collectives requires an address.